docs/source/chatops/notifications.rst

```diff
@@ -178,54 +178,65 @@ task notify is shown below:
         timeout: 180
       on-success: "make_tests"
 
-How do I Setup Notifications for Mistral?
------------------------------------------
+How do I Setup Notifications for Orquesta?
+------------------------------------------
 
-The method for global notifications for the workflow is the same as ActionChain. You have a notify
-section in the action meta when registering. See an
-`example <https://github.com/StackStorm/st2/blob/master/contrib/examples/actions/mistral-basic-two-tasks-with-notifications.yaml#L24>`_.
-Unfortunately, notifications per task are not supported in Mistral as a first class citizen yet.
-This will be added in later releases.
+The Orquesta metadata supports an optional notify input parameter, this specifies the tasks in the workflow that notifications will be generated for. For example:
 
-How do I Skip Notifications for Tasks in a Workflow?
+.. code-block:: yaml
+
+    ---
+    name: notify
+    description: A basic sequential workflow with notify enabled.
+    pack: orquesta_tests
+    runner_type: orquesta
+    entry_point: workflows/sequential.yaml
+    enabled: true
+    parameters:
+      who:
+        required: true
+        type: string
+        default: Stanley
+      notify:
```

Comment:
This is the example used in the unit-tests, but I couldn't get notify in Orquesta working in my setup - but I may be missing something. In the Orquesta pages we just mention there is a notify parameter that we can ignore for now...

Reply:
I got Orquesta notifications working but I also had to add on a notify section as well as the notify parameters, e.g. I also had:
```
notify:
  on-complete:
    routes:
      - slack
    message: "Succeeded"
```

so that it had the details of route etc. 

Do we think the documentation as is is ok, or 
1) should I add in the notify section into the example
2) Should I add in a preceding sentence along following lines:

`The method for specify the route and conditions for when tasks in an Orquesta workflow is the same as ActionChain. You have a notify section in the action meta when registering.   `      

and should I put in a notify section

Reply:
@LindsayHill @armab Having got  notifications working in Orquesta now, I'm wondering if more needs to go into this 
 section on the chatops page. Or alternatively I was mis-understanding something.

Reply:
> @LindsayHill @armab Having got notifications working in Orquesta now, I'm wondering if more needs to go into this
> section on the chatops page. Or alternatively I was mis-understanding something.

So as not to confuse things, I will merge this PR. And generate a new PR with enhancements that I think will make Orquesta notifications even clearer.

Reply:
:+1: step by step